akari/data.py

```
@@ -1,19 +1,39 @@
-from typing import Dict, Generic, TypeVar
+from typing import Any, Dict, Generic, TypeVar
 
 T = TypeVar("T")
 
 
 class AkariDataSetType(Generic[T]):
-    def __init__(self, main: T, others: Dict[str, T]) -> None:
+    def __init__(self, main: T, others: Dict[str, T] = {}) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)